src/napari_workflows/_workflow.py

```diff
@@ -462,7 +485,7 @@ def _get_layer_from_data(viewer: "napari.Viewer", data):
     return None
 
 
-def _generate_python_code(workflow: Workflow, viewer: "napari.Viewer", notebook: bool = False, use_napari:bool = True):
+def _generate_python_code(workflow: Workflow, viewer: Viewer, notebook: bool = False, use_napari:bool = True) -> str:
```

Comment:
I would like to have this not using the Napari class. Idea of "napari.Viewer" as type annotation is that this also works if napari is not installed.

Reply:
Indeed this is a mistake, I should have put quotes around it. Now fixed.